src/extension.ts

```diff
@@ -65,10 +65,18 @@ export function activate(context: vscode.ExtensionContext) {
 	});
 	actionHandler.registerCommand('cosmosDB.attachDatabaseAccount', async () => {
 		const rootNodes = await explorer.getChildren();
-		const attachedNode = <IAzureParentNode<AttachedAccountsTreeItem>>rootNodes.find((node) => node.treeItem instanceof AttachedAccountsTreeItem);
-		if (attachedNode) {
-			await attachedNode.treeItem.attachNewAccount();
-			explorer.refresh(attachedNode);
+		const attachedDatabasesNode = <IAzureParentNode<AttachedAccountsTreeItem>>rootNodes.find((node) => node.treeItem instanceof AttachedAccountsTreeItem);
```

Comment:
Variable name change because I found it a bit unclear whether attachedNode meant the _attached Database_ or the _Attached Database Accounts_ root